File: 1- UI test project/MyFirstJavaExperiment/app/src/main/java/com/alperenavan/myfirstjavaexperiment/MainActivity.java
package com.alperenavan.myfirstjavaexperiment;

import androidx.annotation.IdRes;
import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.ImageView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

    }

    public void changeImage(View view) {

        ImageView imageView = findViewById(R.id.imageView);

        boolean check=true;

        if(check=true){
            imageView.setImageResource(R.drawable.image2);
            check = false;
        }

    }

    public void returnImage(View view){

        ImageView imageView = findViewById(R.id.imageView);

        boolean check = true;

        if(check=true){
            imageView.setImageResource(R.drawable.image1);
            check = false;
        }
    }
}

File: 6-Recipes/app/src/main/java/com/alperenavan/recipes/MainActivity.kt
package com.alperenavan.recipes

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.Menu
import android.view.MenuItem
import androidx.navigation.Navigation
import com.alperenavan.recipes.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {

    private lateinit var binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)



    }

    override fun onCreateOptionsMenu(menu: Menu?): Boolean {

        val menuInflater = menuInflater

        menuInflater.inflate(R.menu.food_add , menu)

        return super.onCreateOptionsMenu(menu)
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {

        if(item.itemId == R.id.food_add_item){
            val action = ListFragmentDirections.actionListFragmentToRecipesFragment()
            Navigation.findNavController(this , R.id.fragmentContainerView).navigate(action)
        }

        return super.onOptionsItemSelected(item)
    }
}